platformio.ini

```diff
@@ -12,3 +12,9 @@
 platform = ststm32
 board = nucleo_l432kc
 framework = arduino
+lib_deps = 
+	br3ttb/PID@^1.2.1
+	pazi88/STM32_CAN@^1.1.2
+	sstaub/Ticker@^4.4.0
+	gitlab-arduino/TimeoutCallback@^1.3.0
+	khoih-prog/STM32_PWM@^1.0.1
```

Comment:
We can remove `khoih-prog/STM32_PWM@^1.0.1` now that we have a solution for FastPWM that doesn't require this library.